frontend/reports/interactions-by-service/interactions-by-service-results.component.tsx

```diff
@@ -162,4 +164,72 @@ export default function InteractionsByService(props) {
       />
     </>
   );
+
+  function getCsvOptions(data): Promise<CsvOptions> {
+    const allPrograms = {
+      Program: "All programs",
+      Service: "---",
+      "Client Count": data.grandTotal.numClients.toLocaleString(),
+      "Interaction Count": data.grandTotal.numInteractions,
+      "Interaction Hours": `"${formatDuration(data.grandTotal.totalDuration)}"`,
```

Comment:
It might be best to not format the duration, since Excel is capable of treating `00:30` as a duration that it can do math equations on, but can't do the same for `0 hours, 30 minutes`

Similar change should be made to every other place in this file that calls formatDuration

```suggestion
      "Interaction Hours": `"${data.grandTotal.totalDuration}"`,
```